src/hooks/useDarkMode.ts

```diff
@@ -1,16 +1,27 @@
 import React, { useEffect, useState } from 'react'
-// React.Dispatch<React.SetStateAction<string>>
+import colourThemes from 'assets/themes.json'
 export default function useDarkMode(): [string, (theme: string) => void] {
   const [theme, setTheme] = useState(localStorage.getItem('carrotTheme') || 'light');
-  const colorTheme = (theme === 'light') ? 'dark' : 'light'
   const themeSet = (theme: string) => {
     setTheme(theme)
     localStorage.setItem('carrotTheme', theme)
   }
   useEffect(() => {
+    const applyTheme = localStorage.getItem('carrotTheme') || 'light'

```

Comment:
you should really make an enum for all the possible localstorage keys by now
```
localStorage.getItem(storage.carrotTheme)
```
or sm